### DISCUSSION TITLE
How to use multipartFetchExchange?

### DISCUSSION THREAD

**Discussion by shamxeed** [urql-graphql/urql#1781] [Q&A] [answered]
I am completely clueless on how to send an image to the server using the `multipartFetchExchange` and there is no example provided in the docs. Anyone here, to help me out, please?

**Comment by JoviDeCroock** [comment, marked as answer]
There's an example: https://github.com/FormidableLabs/urql/blob/main/examples/with-multipart/src/FileUpload.jsx

**Comment by shamxeed** [comment]
Thanks for your quick response, @JoviDeCroock !!!

But I'm a bit confused because I'm using `react-native` but the example shows a web-based version. 
[const [selectedFile, setSelectedFile] = useState();](https://github.com/FormidableLabs/urql/blob/7ec183808b2e766fbe1964c8d1ad6c8bc999d715/examples/with-multipart/src/FileUpload.jsx#L13)
Is the `selectedFile` above be an object with the image data I got from the image-picker?

eg: 
```js
const onPickImage = async () => {
   const result = await ImagePicker.launchImageLibraryAsync({mediaTypes: ImagePicker.MediaTypeOptions.Images})
   if(!result.cancelled) {
      setSelectedFile({
         uri: result.uri,
         name: result.name,
         ........
      })
   }
}
```
Thanks!!

**Reply by JoviDeCroock** [reply]
I think that `result` should be the same as the web-example

**Reply by shamxeed** [reply]
Okay, I'm gon give it a try later in the night and come back to update this question, hopefully, to mark it answered. Thanks a lot once again for being here for me and everyone !!!
